main.go

```diff
@@ -156,8 +168,10 @@ func main() {
 	}
 	go pc.Run(1, stopCh)
 	go pvc.Run(1, stopCh)
+	go nspvc.Run(1, stopCh)
```

Comment:
As the worker goroutine is blocking, might want to increase the number of worker go routines.

Reply:
Let's tune this while testing, see if this could handle all best_practices.